ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml

```diff
@@ -34,6 +34,120 @@ presubmits:
             cpu: 10m
       serviceAccountName: ci-operator
     trigger: ((?m)^/test( all| backend),?(\s+|$))
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/crud
```

Comment:
We should remove crud, new-app, and performance and only have console-e2e

Reply:
@spadgett i know disregard, i thought i was editing the other pr :)

Reply:
@spadgett this is generated by the prowgen tool. it looks like it doesn't remove, just adds. deleted it and regenerated to remove unwanted yaml.